lib/util/utilities.js

```diff
@@ -3,37 +3,59 @@
 const fs = require('fs-extra');
 const path = require('path');
 
-function readAddonEmberVersionFromBower() {
-  let bowerJSONPath = path.join(findAddonPath(), 'bower.json');
-  let bowerJSON = fs.readJsonSync(bowerJSONPath);
+function bowerJsonPath() {
+  return path.join(findAddonPath(), 'bower.json');
+}
+
+function bowerExists() {
+  return fs.existsSync(bowerJsonPath());
+}
 
-  return bowerJSON.dependencies['ember'];
+function readAddonDependencyVersionFromBower(depName) {
+  let bowerJSON = fs.readJsonSync(bowerJSONPath());
+
+  return bowerJSON.dependencies[depName];
 }
 
-function readAddonEmberVersionFromNpm() {
+function readAddonDependencyVersionFromNpm(depName) {
   let packageJSONPath = path.join(findAddonPath(), 'package.json');
   let packageJSON = fs.readJsonSync(packageJSONPath);
 
-  return packageJSON.devDependencies['ember-source'] || packageJSON.dependencies['ember-source'];
+  return packageJSON.devDependencies[depName] || packageJSON.dependencies[depName];
 }
 
 function readAddonEmberVersion() {
-  let emberVersion;
   try {
-    emberVersion = readAddonEmberVersionFromNpm();
+    let emberVersion = readAddonDependencyVersionFromNpm('ember-source');
     if (!emberVersion) {
-      emberVersion = readAddonEmberVersionFromBower();
+      emberVersion = bowerExists() ? readAddonDependencyVersionFromBower('ember') : '*';
     }
+    return emberVersion;
   }
   catch (e) {
-    // catch error
+    console.error(e);
```

Comment:
ESLint is complaining about the use of `console`, thus the failing test. Can you add an ignore line here, to silence it?

Reply:
The previous behavior swallowed errors without emitting them in any way. What's the desired behavior here?